### PR TITLE
chore: add package-lock.json and prettier

### PR DESCRIPTION
**Pull request by frederickfogerty**
## Description

This PR checks in `package-lock.json`, which is recommended to be checked in by npm.

This PR also adds code formatting with Prettier. This is accomplished by adding a `npm run pretty` command which anyone can run to format the source code and tests.

It is also recommended that PrettierCI be added to this repo to enforce code formatting.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "test": "./node_modules/jasmine/bin/jasmine.js",
-    "build": "mkdir -p dist && ./node_modules/browserify/bin/cmd.js ./src/imgix.js > ./dist/imgix.js && ./node_modules/uglify-js/bin/uglifyjs ./dist/imgix.js --mangle > ./dist/imgix.min.js"
+		"build": "mkdir -p dist && ./node_modules/browserify/bin/cmd.js ./src/imgix.js > ./dist/imgix.js && ./node_modules/uglify-js/bin/uglifyjs ./dist/imgix.js --mangle > ./dist/imgix.min.js",
```
Comment (jayeb) [inline package.json]: Lil tab/space war goin on here, huh?

Comment (frederickfogerty) [reply package.json]: A bit! Prettier has different ideas to NPM 🤔

Comment (jayeb) [reply package.json]: We should lock em in a room and have em fight it out

Comment (frederickfogerty) [reply package.json]: I guess yarn will just sit in the corner and eat popcorn

**Pull request merged**